test/components/plots/barPlotTests.ts

```diff
@@ -634,6 +639,11 @@ describe("Plots", () => {
         });
 
       });
+
+      it("vertical() returns false", () => {
+        assert.isFalse(barPlot.vertical());
```

Comment:
Let's put a small message here like Horizontal Bar plots are always not vertical